packages/website/pages/terms.js

```diff
@@ -23,75 +23,158 @@ export default function TermsOfService() {
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '} 
+          will do the following to make the transition for its users as seamless as possible: 
+          (1) give users an export of all data associated with their account, including 
+          the content addresses of this data and information about the Filecoin deals 
+          containing this data, (2) free renewal of Filecoin deals until an automated, 
+          perpetual solution exists, and (3) store a redundant copy of all data on an IPFS 
+          storage provider (e.g., pinning service) with a long-term storage arrangement.
+        </p>
+        <p className="lh-copy">
+          Data that satisfies the data limits are also guaranteed to be stored in Filecoin, 
+          but the time for data to get into a deal and the number of deals are not guaranteed. 
+          Data larger than the limit may not be stored in a Filecoin deal.
         </p>
 
         <h2 className="chicagoflf">
           <HashLink id="data-limits">Data Limits</HashLink>
         </h2>
         <p className="lh-copy">
-          The service accepts arbitrary upload sizes up to 30GB per upload.
-          Currently, the service has no data upload limit. Protocol Labs may
-          amend this term limit at its sole discretion.
+          The service accepts arbitrary upload sizes up to 31GiB per upload. Currently, 
+          accounts do not have a limit to the total data they can upload. {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '} may 
+          amend this limit at its discretion, though this does not affect any data already 
+          uploaded by an account.
         </p>
 
         <h2 className="chicagoflf">
-          <HashLink id="filecoin-deals">Filecoin Deals</HashLink>
+          <HashLink id="cost">Cost</HashLink>
         </h2>
         <p className="lh-copy">
-          Data stored in{' '}
+          {' '}
           <a className="black" href="https://nft.storage">
-            nft.storage
-          </a>{' '}
-          is guaranteed to be available in IPFS. Data may also be stored in
-          Filecoin but the time, duration, and number of deals are not
-          guaranteed.
-        </p>
-        <p className="lh-copy">
-          Deals may be made with Filecoin nodes operating on test network(s)
-          (i.e. not mainnet). These deals will be flagged as not being a mainnet
-          deal and will be lost on network resets. Node’s that are stored data
-          are not guaranteed to be accessible and may only be available via
-          private networks for testing purposes.
+            NFT.Storage
+          </a>{' '} intends to provide free, perpetual storage for NFTs. All data already 
+          uploaded to the service for free will continue to be persisted on the Filecoin 
+          network at no cost to the user until we have built a tool for perpetual, 
```

Comment:
Should not refer to "we" here IMHO.

> until a facility for perpetual, non-custodial storage is available in the Filecoin network.

Although "until" here suggests that it will not be "no cost" at this point.

Reply:
yeah i was struggling with that exact point. took another stab - does that sound better?